### ISSUE TITLE
Unable to upload files using plugin

### ISSUE THREAD

**Issue by fuelen** [gramps-project/gramps-webapi#464]
I've successfully uploaded the tree, but when Gramps tries to upload files I see the error
> Error 500 while connecting to server

In logs I see this error:
> ValueError: '/home/artur/Gramps Media/20211218_185310.jpg' is not in the subpath of '/app/media' OR one path is relative and the other is absolute.

**Comment by fuelen** [comment]
Awesome, this fixed my issue. If this is an expected behaviour, it would be great to guide the user when this error happens.

During the upload, there was an error that 1 file failed to upload, but I don't know which one. I've tried synchronization once again, and I was reported that there is nothing to synchronize. I found only this line in logs
```
ERROR:root:Error parsing list of recent DBs from file /root/.gramps/recent-files-gramps.xml: no element found: line 1, column 0.
```
but I'm not sure if this is related to the error.
